include/seastar/core/memory.hh

```diff
@@ -139,8 +139,35 @@ static constexpr size_t huge_page_size =
 void configure(std::vector<resource::memory> m, bool mbind,
         std::optional<std::string> hugetlbfs_path = {});
 
+/**
```

Comment:
Can't help asking for making this awesome comment into a doxygen-generated documentation (and the one in front of `is_abort_on_allocation_failure` below too).

To check it you can use `ninja -C build/dev docs` and then check the `build/dev/doc/html/namespaceseastar_1_1memory.html` file. Likely it won't render at the first try, because this line sits inside `\cond internal` block, so its worth moving it outside.

Reply:
> Likely it won't render at the first try, because this line sits inside \cond internal block

Right, doing this. 

Question: is whether an API internal or not defined by the this `\cond internal` annotation or whether it appears in an `*::internal` namespace? I would assume the latter, though if the former I guess this would change this API from internal to not?

Reply:
The `*::internal` namespace doesn't affect documentation generation AFAIK. Also, the `*_abort_on_allocation_failure()` is already _out_ of any C++'s `internal` namespace, it only comes under the doxygen's radar, so my request is just to pull it out of there.

Reply:
@xemul - done and checked with `ninja docs`. 

My question was just whether having a method inside the `\cond internal` block had any bearing on whether it was considered an "internal" API for the purposes of stability guarantees, etc. I guess the answer is "no, only the C++ namespace matters".